README.md

```diff
@@ -27,22 +30,21 @@ snippet.
 ```rust
 #[cfg(test)]
 mod test {
-    use testcontainers::clients;
-    use testcontainers::core::WaitFor;
-    use testcontainers::images::generic::GenericImage;
+    use crate::new_nym_client;
     #[tokio::test]
     async fn test_with_nym() {
-        let docker_client = clients::Cli::default();
-        let nym_ready_message = WaitFor::message_on_stderr("Client startup finished!");
-        let nym_dialer_image = GenericImage::new("nym", "latest")
-            .with_env_var("NYM_ID", "test_connection_dialer")
-            .with_wait_for(nym_ready_message.clone())
-            .with_exposed_port(1977);
-        let dialer_container = docker_client.run(nym_dialer_image);
-        let dialer_port = dialer_container.get_host_port_ipv4(1977);
-        let dialer_uri = format!("ws://0.0.0.0:{dialer_port}");
+
+        let nym_id = "test_transport_connection_dialer";
+        #[allow(unused)]
+        let dialer_uri: String;
+        new_nym_client!(nym_id, dialer_uri);
+        todo!("Now you can use the dialer_uri to make requests.")
     }
 }
 ```
 
 One can create as many of these as needed, limited only by the server resources.
+
+**NOTE**: When using a URI for connections, ensure that you're prefixing the
```

Comment:
this might be sort of confusing since `dialer_uri` already has `ws://` in it? I'd just remove this for now lol

Reply:
I also plan on fixing the bug in nym